learntools/sql/ex5.py

```diff
@@ -146,17 +146,24 @@ def check(self, results):
 class TheLongQuery(CodingProblem):
     _var = 'speeds_result'
     def check(self, results):
+        # check 1: check column names
         results.columns = [c.lower() for c in results.columns]
         assert('hour_of_day' in results.columns), ("Your results should have an `hour_of_day` column.")
         assert('num_trips' in results.columns), ("Your results should have an `num_trips` column.")
         assert('avg_mph' in results.columns), ("Your results should have an `avg_mph` column.")
-        if not results.equals(speeds_answer): # keeping because these checks are awesome. may break
-            assert(results.shape[0] == 12), ('You should have 12 rows in your results but have {}.'.format(results.shape[0]))
-            first_hour_num_trips = results.query('hour_of_day == 1').num_trips[0]
-            assert(first_hour_num_trips != 526723), ('You got most of the query right, but forgot to remove rides with `trip_seconds` or `trip_miles` of 0.')
-        assert (results.equals(speeds_answer)), ("The results don't look right. Try again.")
-        
-
+        # check 2: length of dataframe
+        assert(results.shape[0] == speeds_answer.shape[0]), ('You should have {} rows in your results.'.format(speeds_answer.shape[0]))
+        # check 3: particular values
+        hour_to_check = speeds_answer['hour_of_day'][0]
+        # check first value
```

Comment:
These checks are much better than the old checks. But I think we can give more informative error messages.

Not critical before merging, but I think it's easier for users and more trust-inspiring if we say what value we expected and what their answer had.